pom.xml

```diff
@@ -5,60 +5,55 @@
 	<name>Snowstorm</name>
 	<description>SNOMED CT Terminology Server Using Elasticsearch</description>
 
-	<groupId>org.snomed</groupId>
 	<artifactId>snowstorm</artifactId>
 	<version>8.2.0-SNAPSHOT</version>
 	<parent>
-		<groupId>org.springframework.boot</groupId>
-		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>2.7.0</version>
-		<!-- Remember to update documentation links if this version changes -->
+		<groupId>org.snomed</groupId>
+		<artifactId>snomed-parent-bom</artifactId>
+		<version>1.0.4-SNAPSHOT</version>
 	</parent>
 
 	<properties>
-		<java.version>11</java.version>
-		<maven.compiler.source>${java.version}</maven.compiler.source>
-		<maven.compiler.target>${java.version}</maven.compiler.target>
-		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<packageName>${project.artifactId}</packageName>
 		<!--
 		Current Elasticsearch _Server_ version supported by AWS is 7.10.0
 		N.B. Remember to keep TestConfig.ELASTIC_SEARCH_SERVER_VERSION and getting-started guide updated.
 		-->
-		<elasticvc.version>3.3.0</elasticvc.version>
-		<elasticsearch.version>7.10.0</elasticsearch.version>
-
-		<spring-boot.version>${project.parent.version}</spring-boot.version>
-		<snomed-boot.version>4.0.2-SNAPSHOT</snomed-boot.version>
-		<spring-cloud.version>2021.0.3</spring-cloud.version>
-
-		<!-- FHIR HAPI library -->
-		<hapi.version>5.6.3-si</hapi.version>
-
-		<docker.image.prefix>snomedinternational</docker.image.prefix>
-		<slf4j.version>1.7.36</slf4j.version>
-		<log4j.version>2.17.0</log4j.version>
-		<aws-java-sdk.version>1.11.221</aws-java-sdk.version>
+		<snomed-owl-toolkit.version>3.1.0-SNAPSHOT</snomed-owl-toolkit.version>
```

Comment:
This line is not needed. It's needed in SnowstormX because that uses an experimental owl-toolkit version.